modules/em/include/DensityMap.h

```diff
@@ -373,6 +373,21 @@ class IMPEMEXPORT DensityMap : public IMP::Object {
       \return the new cropped map.
    */
   DensityMap *get_cropped(float threshold);
+  
+  //! Create and return a new cropped map based on particles
+  /** The map is cropped based on voxel proximity to a set of particles.
+      All voxel centers farther than [distance] away from any
+      particle center in [ps] will have their density value set to 0.0.
+      
+      setting inverse to true will set all voxel centeres within
+      [distance] of a particle to 0.0.
+
+      \param[in] ps List of particles used to crop map
+      \param[in] distance Distance from particles at which to crop map
+      \param[in] inverse Set to true to crop the particle volume of the map
+      \return the new cropped map.
+   */
+  DensityMap *get_cropped(Particles ps, float distance, bool inverse = false);
```

Comment:
We generally (although not always) use `double` for floating-point arguments. There is little reason to use plain `float` on a modern machine.